### PR TITLE
Add host details view procedure

### PR DESCRIPTION
**Pull request by bangelic**
Created a procedure to view the host details page from a content host.
The procedure involves going to the Administer
settings.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.5/Katello 4.7 (planned Satellite 6.13)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only)
* [ ] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [ ] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8, orcharhino 6.1 on EL7, orcharhino 6.2 on EL7/8)
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

File: guides/common/modules/proc_viewing-host-details-from-a-content-host.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[id="Viewing_Host_Details_from_a_Content_Host_{context}"]
+= Viewing Host Details from a Content Host
+
+Use this procedure to view the host details page from a content host.
+
+.Procedure
+
```
Comment (maximiliankolb) [inline guides/common/modules/proc_viewing-host-details-from-a-content-host.adoc:7]:
```suggestion
```

**Review by maximiliankolb — COMMENTED** [review]
I am unsure about the first list item. Is this maybe a prerequisite?

**Comment by maximiliankolb** [comment]
@bangelic Off-topic: What's your handle on https://community.theforeman.org/? We have a group to which I'd like to invite you: https://community.theforeman.org/g/docs

**Comment by bangelic** [comment]
> @bangelic Off-topic: What's your handle on https://community.theforeman.org/? We have a group to which I'd like to invite you: https://community.theforeman.org/g/docs

bangelic

**Review by vsedmik — APPROVED** [review]
LGTM

File: guides/common/modules/proc_viewing-host-details-from-a-content-host.adoc [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+. In the {ProjectWebUI}, navigate to *Hosts* > *Content Hosts*
+. Click the content host you want to view.
+. Select the *Details* tab to see the host details page.
+
+The cards in the *Details* tab show details for the *System properties*, *BIOS*, *Networking interfaces*, *Operating system*, *Provisioning templates*, and *Provisioning*.
+Registered content hosts show additional cards for *Registration details*, *Installed products*, and *HW properties* providing information about *Model*, *Number of CPU(s)*, *Sockets*, *Cores per socket* and *RAM*.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_viewing-host-details-from-a-content-host.adoc:13]:
```suggestion
Registered content hosts show additional cards for *Registration details*, *Installed products*, and *HW properties* providing information about *Model*, *Number of CPU(s)*, *Sockets*, *Cores per socket*, and *RAM*.
```

**Pull request merged**